vision/src/autogluon/vision/detector/detector.py

```diff
@@ -49,16 +49,16 @@ def path(self):
     def fit(self,
             train_data,
             tuning_data=None,
-            time_limit=None,
+            time_limit=7200,
```

Comment:
We may want to have this be 'default' or 'auto' (Which is then set to 7200) so we can specifically log to the user that they should specify a time limit themselves if the default time is not what they want.